README.md

```diff
@@ -29,28 +29,9 @@ and many others.
 |**Linux**|[![Linux Debug](https://msrcambridge.visualstudio.com/Infer.NET/_apis/build/status/Nightly%20Linux%20Debug)](https://msrcambridge.visualstudio.com/Infer.NET/_build/latest?definitionId=137)|[![Linux Release](https://msrcambridge.visualstudio.com/Infer.NET/_apis/build/status/Nightly%20Linux%20Release)](https://msrcambridge.visualstudio.com/Infer.NET/_build/latest?definitionId=136)|
 |**macOS**|[![macOS Debug](https://msrcambridge.visualstudio.com/Infer.NET/_apis/build/status/Nightly%20macOS%20Debug)](https://msrcambridge.visualstudio.com/Infer.NET/_build/latest?definitionId=139)|[![macOS Release](https://msrcambridge.visualstudio.com/Infer.NET/_apis/build/status/Nightly%20macOS%20Release)](https://msrcambridge.visualstudio.com/Infer.NET/_build/latest?definitionId=138)|
 
-## Installation
+## Installing pre-built binaries
 
-Infer&#46;NET runs on Windows, Linux, and macOS - any platform where [.NET Core 2.1](https://github.com/dotnet/core) is available.
-
-First ensure you have installed [.NET Core 2.1](https://www.microsoft.com/net/download/). Infer&#46;NET also works on the .NET Framework 4.6.1 and above.
-
-Once you have an app, you can install the Infer&#46;NET NuGet package(s) from the .NET Core CLI using:
-```
-dotnet add package Microsoft.ML.Probabilistic
-dotnet add package Microsoft.ML.Probabilistic.Compiler
-dotnet add package Microsoft.ML.Probabilistic.Learners
-```
-
-or from the NuGet package manager:
-```powershell
-Install-Package Microsoft.ML.Probabilistic
-Install-Package Microsoft.ML.Probabilistic.Compiler
-Install-Package Microsoft.ML.Probabilistic.Learners
-Install-Package Microsoft.ML.Probabilistic.Visualizers.Windows
-```
-
-Or alternatively you can add the Microsoft.ML.Probabilistic.* package(s) from within Visual Studio's NuGet package manager or via [Paket](https://github.com/fsprojects/Paket).
+Binaries for Infer.NET are located on [nuget.org](https://www.nuget.org/packages?q=Microsoft.ML.Probabilistic).  These binaries are cross-platform and work anywhere that .NET is supported---there is no need to select your platform.  You do not need to clone the GitHub repository to use the pre-built binaries.
```

Comment:
Github seems to render `---` verbatim: ---
You can use symbol itself: `—`, or is html escape sequence: `&mdash;`
— &mdash;